crates/polars-sql/Cargo.toml

```diff
@@ -16,6 +16,7 @@ polars-lazy = { workspace = true, features = ["abs", "binary_encoding", "concat_
 polars-plan = { workspace = true }
 
 hex = { workspace = true }
+once_cell = { workspace = true }
```

Comment:
We can use https://doc.rust-lang.org/std/sync/struct.OnceLock.html now. The `once_cell` crate has become redundant. We need to get that out.

Reply:
Looks like we're using `once_cell::sync::Lazy` in quite a few places (17 according to a quick grep through the code) - if the replacement pattern is `LazyLock` that hasn't been stabilised yet 🤔

Reply:
No, this is what I meant. https://doc.rust-lang.org/std/sync/struct.OnceLock.html

We use that on a few places already. You create a `OnceLock` and a function `get_locked` which gets or intits.

Reply:
Ahh, nice; will update later this evening 👌

Reply:
Done 👌